components/ExcludeFromIndex.tsx

```diff
@@ -0,0 +1,7 @@
+export default function ExcludeFromIndex({ children }: any) {
+  return (
+    <div key="excludeFromIndex" className="exclude-from-index">
```

Comment:
nie trzeba nam chyba propa key, bo zwracamy jedną instancję.

a poza tym, czy dodanie diva nie zmieni layoutu? może lepiej dodać klasę do poszczególnych elementów?

```suggestion
    <div className="exclude-from-index">
```

Reply:
Z tego co sprawdzałem, layout się nie zmienił. Zacząłęm od dodawania klasy do poszczególnych elementów, ale jest to bardziej manualny proces i jeśli zdecydujemy kiedyś zmienić nazwę klasy na inną to trzeba to będzie zamieniać w każdym miejscu. Jednak możemy wrócić do tego rozwiązania, co uważasz? Myślę, że nie będziemy tego zmieniać za często.

Reply:
Okej, jeśli layout jest nienaruszony, to możesz mergować. 👌

Reply:
Jednak zmieniam na dodawanie klasy :D Dzięki temu nie będziemy ingerować w strukturę strony dodatkowym divem. Tak chyba będzie lepiej.